sweetalert2.d.ts

```diff
@@ -227,7 +227,13 @@ declare module 'sweetalert2' {
         'text' | 'email' | 'password' | 'number' | 'tel' | 'range' | 'textarea' | 'select' | 'radio' | 'checkbox' |
         'file' | 'url' | undefined;
 
-    export type SweetAlertDismissalReason = 'cancel' | 'close' | 'overlay' | 'esc' | 'timer';
+    export enum SweetAlertDismissReason {
+        'CANCEL' = 'cancel',
```

Comment:
You don't need the `'` quotes on the left-hand.

Reply:
Done